lib/mayaUsd/fileio/shading/shadingModeExporterContext.cpp

```diff
@@ -300,12 +300,20 @@ UsdMayaShadingModeExportContext::GetAssignments() const
         MDagPath dagPath = allDagPaths[instanceNumber];
         TF_VERIFY(dagPath.instanceNumber() == instanceNumber);
 #endif
-
         MFnDagNode dagNode(dagPath, &status);
         if (!status) {
             continue;
         }
 
+        ret.hasAnyAssignment = true;
+
+        // Note: the connections will be to the mesh, but the selection often
+        //       contains the transform above the mesh or a higher-up ancestor,
+        //       so also check the ancestors of the mesh.
+        for (MDagPath walkUpPath(dagPath); walkUpPath.length() > 0; walkUpPath.pop())
+            if (GetExportArgs().fullObjectList.hasItem(walkUpPath))
```

Comment:
Is there a policy in code guidelines for one line if cases like if they should have {} or not? I couldn't find one. This is minor, anyways

Reply:
There is no policy for this (no {} is allowed), other than the formatting done by clang-format.